Automattic/wp-calypso
# Removes My Home card for DIFM Lite in-progress sites

#### Changes proposed in this Pull Request

* Removes the My Home card for DIFM Lite in-progress sites which was added in #56313
* This card was already hidden when #56935 was shipped, this change deletes this redundant code.

#### Testing instructions



* Go to My Home and confirm that there are no JS errors.



Related backend diff: D70643-code
